cadc-storage-permissions-client/src/main/java/org/opencadc/inventory/permissions/PermissionsClient.java

```diff
@@ -114,11 +114,10 @@ public PermissionsClient(URI serviceID) {
      *
      * @param artifactURI Identifies the artifact for which to retrieve grant information.
      * @return The read grant information.
-     * 
-     * @throws ResourceNotFoundException If the file could not be found.
+     *
```

Comment:
The possible return of null needs to be documented in the @return since it is meaningful  and must be interpretted (and in getWriteGrant)

Reply:
Added documentation that a null could be returned.